subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/TestsFilter.kt

```diff
@@ -9,19 +9,19 @@ import com.avito.android.test.annotations.UIComponentStub
 import com.avito.android.test.annotations.UIComponentTest
 
 private val manualAnnotations = setOf(
-    ManualTest::javaClass.name,
-    UIComponentStub::javaClass.name,
-    E2EStub::javaClass.name
+    ManualTest::class.java.name,
```

Comment:
I don't know a reason.
I ran the instrumentationDynamic and see `javaClass` but the annotation class name
<img width="559" alt="Screenshot 2021-01-22 at 21 15 29" src="https://user-images.githubusercontent.com/8560287/105529138-037af780-5cf7-11eb-8039-90de1741d53c.png">

And my hypothesis that `E2EStub::javaClass.name` works incorrectly without dependency on the kotlin-reflection